hypertrace-graphql-metric-schema/src/main/java/org/hypertrace/graphql/metric/request/DefaultMetricAggregationRequestBuilder.java

```diff
@@ -70,12 +72,18 @@
         context, requestScope, metricQueryableFieldStream, this::build);
   }
 
-  @Override
   public MetricAggregationRequest build(
-      AttributeModel attribute,
-      AttributeModelMetricAggregationType aggregationType,
-      List<Object> arguments) {
-    return new DefaultMetricAggregationRequest(attribute, aggregationType, arguments);
+          AttributeModel attribute,
+          AttributeModelMetricAggregationType aggregationType,
+          List<Object> arguments) {
+    return new DefaultMetricAggregationRequest(attribute, aggregationType, arguments, Collections.EMPTY_LIST);
```

Comment:
```suggestion
    return new DefaultMetricAggregationRequest(attribute, aggregationType, arguments, Collections.emptyList());
```

`EMPTY_LIST` returns a raw list - I can't think of any times `emptyList()` wouldn't be preferable.

Reply:
This is also the issue I was referring to on the interface - any calls to this api would silently omit baselines on the response.